src/api/emails/index.ts

```diff
@@ -89,6 +89,32 @@ const securityCodeEmail = async (emailAddress: string, firstName: string, securi
   }
 };
 
+/**
+ * passwordResetLink
+ * Send "reset your password" email to an exporter
+ * @param {String} Email address
+ * @param {String} First name
+ * @param {String} Password reet token
+ * @returns {Object} callNotify response
+ */
+const passwordResetLink = async (emailAddress: string, firstName: string, passwordResetHash: string): Promise<EmailResponse> => {
+  try {
+    console.info('Sending email for account password reset');
+
+    const templateId = EMAIL_TEMPLATE_IDS.ACCOUNT.PASSWORD_RESET;
+
+    const variables = { firstName, passwordResetToken: passwordResetHash };
+
+    const response = await callNotify(templateId, emailAddress, variables);
+
+    return response;
+  } catch (err) {
+    console.error(err);
+
+    throw new Error(`Sending email for account password reset ${err}`);
```

Comment:
this file is getting quite big.  tech debt to split this into folders and have one index file with imports? Like custom-resolvers?